site/content/docs/user/known-issues.md

```diff
@@ -248,6 +249,17 @@ open /home/user/.docker/config.json: permission denied
 To fix this problem, either follow the docker's docs [manage docker as a non root user][manage docker as a non root user],
 or try to use `sudo` before your commands (if you get `command not found` please check [this comment about sudo with kind][sudo with kind]).
 
+
+## Docker on Windows
+
+[Docker Desktop for Windows][docker desktop for windows] supports running both Linux (the default) and Windows Docker containers.
+
+`kind` will fail create a cluster, if your Docker Desktop for Windows using `Windows containers` mode.
+
+This is because `kind` only distributed docker images base on Linux.
```

Comment:
Only supports linux "node" containers while Windows, Mac, and Linux _host_ machines are supported. Windows node support is not currently feasible due to how windows containers work.